package.json

```diff
@@ -37,7 +37,7 @@
   ],
   "dependencies": {
     "deep-equal": "^1.1.0",
-    "eth-block-tracker": "^4.4.2",
+    "eth-block-tracker": "^5.0.1",
```

Comment:
Any idea why this update was needed? I would have expected v4 of eth-block-tracker to work on v14.